src/components/Slider/helpers.js

```diff
@@ -0,0 +1,25 @@
+import { getPercent } from '../../helpers/common';
+
+export function createSlidesXPoints(
+  sliderContainer,
+  sliderContent,
+  slidesCount
+) {
+  //prevent console errors
+  if (!sliderContainer || !sliderContent) return null;
+
+  return new Array(slidesCount).fill().map((_, i) => {
```

Comment:
I think you don't need to fill array here, since after new Array() it already filled with undefined values. You basically doing fill with map anyway

Reply:
btw, maybe you want to try to bring here lodash/fp and use flow() ? 😏

Reply:
It works without "fill" locally but after build it crashes...
I don't think that we need the whole lodash library for that, but I could write my own realization of flow for that case :)

Reply:
I believe it (lodash) should have a tree shaking model, so importing the lib doesn't mean to include it into the bundle. In the end, you will come up with the more less the same code )